src/cephadm/cephadm

```diff
@@ -4872,6 +4872,13 @@ def finish_bootstrap_config(
             'restart',
             get_unit_name(fsid, 'mon', mon_id)
         ])
+    elif 'image' in ctx and ctx.image:
+        # we still want to assimilate the given container image if provided
+        cli([
+            'config', 'set', 'global', 'container_image', f'{ctx.image}'
+        ], {
+            mon_dir: '/var/lib/ceph/mon/ceph-%s:z' % mon_id
+        })
```

Comment:
```suggestion
        cli(['config', 'set', 'global', 'container_image', f'{ctx.image}'])
```

do we need the mon dir as an extra mount?

Reply:
nope, removing